ttnn/ttnn/device.py

```diff
@@ -72,4 +73,14 @@ def dump_device_memory_state(device, prefix=""):
     ttl.device.DumpDeviceMemoryState(device, prefix)
 
 
+def is_wormhole_b0():
```

Comment:
please change this function to take in `device` and figure out the arch from the device. Because what would happen if we have a heterogeneous system?